test/end-to-end/pages/payment/paymentbreakdown.js

```diff
@@ -1,11 +1,19 @@
 'use strict';
 
-module.exports = async function() {
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const paymentContentEn = require('app/resources/en/translation/payment/breakdown');
+const paymentContentCy = require('app/resources/cy/translation/payment/breakdown');
+const testConfig = require('config');
+
+module.exports = async function(language = 'en') {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
+    const paymentContent = language === 'en' ? paymentContentEn : paymentContentCy;
 
     await I.checkPageUrl('app/steps/ui/payment/breakdown');
-    await I.waitForElement({css: 'form[action="/payment-breakdown"]'});
-    const locator = {css: '.govuk-button'};
-    await I.waitForElement(locator);
-    await I.navByClick(locator);
+    await I.wait(3);
```

Comment:
Are there no explicit waits we can do to remove this?

Reply:
I tried very hard to avoid this, but even using other explicit waits resulted in the step hanging here until a step retry was triggered, which meant waiting around 40seconds. So this is actually the faster option for now, until the underlying source of the "Execution context was destroyed, most likely because of a navigation." error for Puppeteer is understood. 
I'll raise a ticket to dig into this error further, but at the moment this is the lesser of the 2 evils.

Reply:
ok!